stories/AuctionContainer.tsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+import StoryRouter from 'storybook-router'
+import { makeProviderDecorator, storeInit } from './helpers'
+
+import AuctionContainer from 'components/AuctionContainer'
+import AuctionFooter from 'components/AuctionFooter'
+import AuctionHeader from 'components/AuctionHeader'
+import AuctionPriceBar from 'containers/AuctionPriceBar'
+import AuctionProgress from 'components/AuctionProgress'
+import AuctionSellingGetting from 'components/AuctionSellingGetting'
+import AuctionStatus, { Status } from 'components/AuctionStatus'
+import AuctionWalletSummary from 'components/AuctionWalletSummary'
+import ButtonCTA from 'components/ButtonCTA'
+import TokenPair from 'containers/TokenPair'
+
+import { action } from '@storybook/addon-actions'
+import { boolean, number, text } from '@storybook/addon-knobs'
+
+import { TokenCode } from 'types'
+
+const Provider = makeProviderDecorator(storeInit())
+
+storiesOf('AuctionContainer', module)
+  .addDecorator(StoryRouter())
+  .addDecorator(Provider)
+  .addWithJSX('PAGE 2', () =>
+    <AuctionContainer auctionDataScreen="amount">
+      <AuctionHeader 
```

Comment:
Any reason not to
```
<AuctionHeader backTo="/" >
   Token Auction ETH/GNO
< /AuctionHeader>
```
?

Reply:
didn't like that style when writing it. No other pref either way though

Reply:
good then